### DISCUSSION TITLE
Can not open link generate by success rate

### DISCUSSION THREAD

**Discussion by OrezzerO** [apache/skywalking#12209] [General]
## Reproduction: 
1. Open [Live Demo](http://demo.skywalking.apache.org/dashboard/GENERAL/Service/YWdlbnQ6OmFwcA==.1/General-Service) 
2. Generate link of success rate, then get the link
```
demo.skywalking.apache.org/page/GENERAL/Service/YWdlbnQ6OmFwcA==.1/null/null/null/null/null/%7B%22type%22%3A%22Widget%22%2C%22graph%22%3A%7B%22type%22%3A%22Line%22%2C%22step%22%3Afalse%2C%22smooth%22%3Afalse%2C%22showSymbol%22%3Atrue%2C%22showXAxis%22%3Atrue%2C%22showYAxis%22%3Atrue%7D%2C%22metricConfig%22%3A%5B%7B%22label%22%3A%22success_rate%22%7D%5D%2C%22height%22%3A280%2C%22expressions%22%3A%5B%22service_sla%2F100%22%5D%2C%22widget%22%3A%7B%22title%22%3A%22Success%2520Rate%2520(%2525)%22%2C%22tips%22%3A%22%22%7D%2C%22auto%22%3A1800000%2C%22autoPeriod%22%3A6%7D
```
This link can not show data successfully. 

After url decoding, it's like
```
demo.skywalking.apache.org/page/GENERAL/Service/YWdlbnQ6OmFwcA==.1/null/null/null/null/null/{"type":"Widget","graph":{"type":"Line","step":false,"smooth":false,"showSymbol":true,"showXAxis":true,"showYAxis":true},"metricConfig":[{"label":"success_rate"}],"height":280,"expressions":["service_sla/100"],"widget":{"title":"Success%20Rate%20(%25)","tips":""},"auto":1800000,"autoPeriod":6}
```

I guess it's a bug causing by special character, so i try to delete `(%25)` ( `(%2525)`),  it works. 

Modified link: 
```
demo.skywalking.apache.org/page/GENERAL/Service/YWdlbnQ6OmFwcA==.1/null/null/null/null/null/%7B%22type%22%3A%22Widget%22%2C%22graph%22%3A%7B%22type%22%3A%22Line%22%2C%22step%22%3Afalse%2C%22smooth%22%3Afalse%2C%22showSymbol%22%3Atrue%2C%22showXAxis%22%3Atrue%2C%22showYAxis%22%3Atrue%7D%2C%22metricConfig%22%3A%5B%7B%22label%22%3A%22success_rate%22%7D%5D%2C%22height%22%3A280%2C%22expressions%22%3A%5B%22service_sla%2F100%22%5D%2C%22widget%22%3A%7B%22title%22%3A%22Success%2520Rate%2520%22%2C%22tips%22%3A%22%22%7D%2C%22auto%22%3A1800000%2C%22autoPeriod%22%3A6%7D
```

**Comment by wu-sheng** [comment]
I generated the link as

[demo.skywalking.apache.org/page/GENERAL/Service/YWdlbnQ6OmFwcA==.1/null/null/null/null/null/%7B%22type%22%3A%22Widget%22%2C%22graph%22%3A%7B%22type%22%3A%22Card%22%2C%22fontSize%22%3A14%2C%22textAlign%22%3A%22center%22%2C%22showUnit%22%3Atrue%7D%2C%22metricConfig%22%3A%5B%7B%22unit%22%3A%22%2525%22%7D%5D%2C%22height%22%3A160%2C%22expressions%22%3A%5B%22avg(service_sla)%2F100%22%5D%2C%22widget%22%3A%7B%22title%22%3A%22Success%2520Rate%22%2C%22tips%22%3A%22%22%7D%2C%22auto%22%3A1800000%2C%22autoPeriod%22%3A6%7D](http://demo.skywalking.apache.org/page/GENERAL/Service/YWdlbnQ6OmFwcA==.1/null/null/null/null/null/%7B%22type%22%3A%22Widget%22%2C%22graph%22%3A%7B%22type%22%3A%22Card%22%2C%22fontSize%22%3A14%2C%22textAlign%22%3A%22center%22%2C%22showUnit%22%3Atrue%7D%2C%22metricConfig%22%3A%5B%7B%22unit%22%3A%22%2525%22%7D%5D%2C%22height%22%3A160%2C%22expressions%22%3A%5B%22avg(service_sla)%2F100%22%5D%2C%22widget%22%3A%7B%22title%22%3A%22Success%2520Rate%22%2C%22tips%22%3A%22%22%7D%2C%22auto%22%3A1800000%2C%22autoPeriod%22%3A6%7D)

And opened it successfully on my Mac Chome.

**Reply by OrezzerO** [reply]
I am using mac chrome too, but it generate the link contains `(%2525)`. I have disable cache and refresh the page:    
![image](https://github.com/apache/skywalking/assets/13081689/036d3c69-c410-4ee7-acfc-d7a9000122a9)

Then i use safari to try, same problem: 
<img width="1280" alt="image" src="https://github.com/apache/skywalking/assets/13081689/be0edfc4-364c-4864-9d10-64aa0e6df22a">

**Reply by OrezzerO** [reply]
I mean `Success Rate (%)`  line graph, not  `Success Rate`

**Reply by wu-sheng** [reply]
Which page we have this?

**Reply by OrezzerO** [reply]
<img width="1720" alt="image" src="https://github.com/apache/skywalking/assets/13081689/ff0f8320-7d18-4516-bd46-05a651781617">
This page :http://demo.skywalking.apache.org/dashboard/GENERAL/Service/YWdlbnQ6OmFwcA==.1/General-Service

**Reply by wu-sheng** [reply]
@Fine0830 Please check this.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking-booster-ui/pull/394
